### PR TITLE
bump mysql version number

### PR DESCRIPTION
**Pull request by samanthajayasinghe**

### CODE REVIEW

File: installer/environmentCheck/system_requirements.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ phpversion:
 
 mysqlversion:
     min: '5.5'
-    max: '5.7.26'
+    max: '5.7.28'
```
Comment (RajithaKumara) [inline installer/environmentCheck/system_requirements.yml]: Can we upgrade to 5.7.29 if supported.

Comment (samanthajayasinghe) [reply installer/environmentCheck/system_requirements.yml]: done

Comment (samanthajayasinghe) [reply installer/environmentCheck/system_requirements.yml]: @RajithaKumara can you merge this PR ?

File: installer/environmentCheck/system_requirements.yml [hunk 2]
```diff
@@ -1,11 +1,11 @@
 phpversion:
     min: '7.0'
-    max: '7.3.6'
+    max: '7.3.13'
```
Comment (RajithaKumara) [inline installer/environmentCheck/system_requirements.yml]: PHP 7.3.15 also released.

Comment (samanthajayasinghe) [reply installer/environmentCheck/system_requirements.yml]: done

**Pull request merged**